modules/update-best-practices.adoc

```diff
@@ -43,11 +43,6 @@ OpenShift SDN network plugin was deprecated in versions 4.15 and 4.16. With this
 
 If your {product-title} cluster is still using the OpenShift SDN CNI, see link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.16/html/networking/ovn-kubernetes-network-plugin#migrate-from-openshift-sdn[Migrating from the OpenShift SDN network plugin].
 
-[IMPORTANT]
-====
-It is not possible to update a cluster to {product-title} 4.17 if it is using the OpenShift SDN network plugin. You must migrate to the OVN-Kubernetes plugin before upgrading to {product-title} 4.17. 
```

Comment:
You might want to confirm with SME/QE (when seeking approval) if the note should now say 4.18 instead of 4.17 instead of removing. But if this is something already confirmed, please ignore this comment.

Reply:
@diksha1999 If the update is from 4.17 -> 4.18 then it is not necessary to mention Openshift SDN plugin as 4.17 the only supported plugin in OVN-Kubernetes .  So note can be removed. If the customer is upgrading from 4.16 -> 4.17 -> 4.18 then this note is relevant. So you may want to rephrase the note in this case.

Reply:
Hi @asood-rh . Just to confirm that 4.19 and onwards should have no mention whatsoever of SDN?

Compare the following version doc files where one version contains the "SDN support removal" section and the 4.19 version of the file removes it):

4.18: https://github.com/openshift/openshift-docs/blob/enterprise-4.18/modules/update-best-practices.adoc#sdn-support-removal

4.19: https://github.com/openshift/openshift-docs/blob/enterprise-4.19/modules/update-best-practices.adoc#sdn-support-removal